docs/guides/developer/docs/develop/frontends/overview.md

```diff
@@ -0,0 +1,33 @@
+Frontends
+---------
+
+Opencast boasts multiple graphical user interfaces for different purposes, living in different modules or even projects.
+The following is a brief overview.
+
+### Admin UI
+- Lives in the `admin-ui-frontend` module.
+- The main frontend for managing Opencast.
+- Style guide can be found [here](./admin-ui/development.md).
+
+### New Admin UI
+- Lives in its [own GitHub repository](https://github.com/opencast/opencast-admin-interface). Builds end up in the
+  `admin-ui-interface` module.
+- Will eventually become and replace the (old) Admin UI.
```

Comment:
```suggestion
- Is the current default admin UI starting with 15.x
```

Reply:
I still think mentioning the replace bit to quickly make clear what the relation between the old and new ui is is worthwhile. Maybe something like "The default Admin UI starting with 15.x, to eventually replace the (old) Admin UI"?